vision_evaluation/evaluators.py

```diff
@@ -816,3 +817,168 @@ def __init__(self):
         super().__init__(metric='SPICE')
         self.predictions = []
         self.targets = []
+
+
+class MattingEvaluatorBase(Evaluator):
+    """
+    Base class for image matting evaluator
+    """
+    def __init__(self):
+        super(MattingEvaluatorBase, self).__init__()
+        self._metric = None
+        self._num_samples = 0
+        self._metric_sum = 0
+
+    def reset(self):
+        super(MattingEvaluatorBase, self).reset()
+        self._num_samples = 0
+        self._metric_sum = 0
+
+    def _convert2binary(self, mask, threshold=128):
```

Comment:
threshold not used.

Reply:
resolved by changing the hard encoded value 128 in the `_convert2binary` function to threshold!